README.md

```diff
@@ -1,5 +1,5 @@
 # morph
-[![Build Status](https://travis-ci.org/DBCDK/morph.svg?branch=master)](https://travis-ci.org/DBCDK/morph)
+[![Build](https://github.com/DBCDK/morph/actions/workflows/build.yaml/badge.svg)](https://github.com/DBCDK/morph/actions/workflows/build.yaml)
```

Comment:
Hmm, this URL has no reference to any branch. 

This made me think it has some sort of notion of "last run of workflow" across all branches, and indeed, look, I broke it:

https://github.com/DBCDK/morph/actions/workflows/build.yaml/badge.svg

I simply created a broken branch here:

https://github.com/DBCDK/morph/tree/break-ci-pipeline

which is (right now...) the latest run